### DISCUSSION TITLE
Account blocked

### DISCUSSION THREAD

**Discussion by AmiXady** [codewars/codewars.com#3129] [General]
What to do if your account is blocked? I wrote to the general codewars email, but they didn’t respond. It seems they all died there

**Comment by hobovsky** [comment]
Activities of moderation team are currently limited due to the Christmas Holiday break, possibly until New Year. Your email will be eventually picked up, but you need to be patient.

**Reply by AmiXady** [reply]
wrote codewars via email. It seems they were still drunk after the new year. I still didn't understand what they answered. The only noticeable message was to appreciate their maintenance.

**Reply by AmiXady** [reply]
> Activities of moderation team are currently limited due to the Christmas Holiday break, possibly until New Year. Your email will be eventually picked up, but you need to be patient.


I thought it was temporary, but turns out my account still hasn’t been restored.
Though I’ve already abandoned it, so it doesn’t really matter.
